aws/lambda/python/sandbox_cleanup.py

```
@@ -0,0 +1,84 @@
+import json
+import boto3
+import time
```

Ruff (0.8.2)

`time` imported but unused

Remove unused import: `time`

(F401)